ra2ce/analysis/losses/losses_base.py

```diff
@@ -97,10 +97,8 @@ def __init__(
         self.part_of_day: PartOfDayEnum = self.analysis.part_of_day
         self.analysis_type = self.analysis.analysis
         self.duration_event: float = self.analysis.duration_event
-        self.hours_per_day: float = self.analysis.hours_per_day
-        self.production_loss_per_capita_per_hour = (
-            self.analysis.production_loss_per_capita_per_day / self.hours_per_day
-        )
+        self.hours_per_day: float = 24  # 24 hours in a day
```

Comment:
```suggestion
        self.hours_per_day: float = self.analysis.hours_per_day  # 24 hours in a day as a default value
```
According to Remark 1.

Reply:
Agreed